kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/InstallKT.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kubernetes.operator;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InstallKT {
```

Comment:
Possibly not very important but `*.kt` is the extension for Kotlin src files so a `KT` suffix for Kube Test might be a bit of a surprise to readers.

Reply:
I don't think that's important. Class names and file extensions are different things. I don't think you'd complain about a classname that ended in `Doc` but had nothing to do with Word.